pkg/app/log.go

```diff
@@ -1,15 +1,15 @@
 package app
 
 import (
-	"bytes"
 	"fmt"
 	"strings"
 	"time"
 
-	log "github.com/sirupsen/logrus"
+	log "github.com/flant/shell-operator/pkg/unilogger"
```

Comment:
Why is the same package imported under two different names?

Reply:
https://github.com/flant/shell-operator/pull/671#discussion_r1808806772 answered

Reply:
fixed